domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/response/TimelineEvent.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.hibernate.validator.constraints.NotBlank;
+import java.util.Objects;
+import javax.validation.constraints.Size;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class TimelineEvent {
+
+    @NotBlank
+    @Size(max = 25)
+    private final String date;
+
+    @NotBlank
+    @Size(max = 99000)
+    private final String description;
+
+    public TimelineEvent(final String date, final String description) {
```

Comment:
Please remove final from construct args, re-assignement is blocked by Checkstyle rule.